server.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, jsonify, request
+from models import db, ACMRequest
+from settings import MYSQL
+from sqlalchemy.sql.expression import func, text
+from datetime import datetime
+import logging
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://{}:{}@{}/{}'.format(
+    MYSQL['user'],
+    MYSQL['password'],
+    MYSQL['host'],
+    MYSQL['dbname']
+)
+
+db.init_app(app)
+db.create_all(app=app)
+
+@app.route('/request/', methods=['GET'])
```

Comment:
I think we don't want to have trailing slashes on any of these request routes.

i.e. `/request/` should be `/request`